pykit/transport/http/HttpConnection.py

```diff
@@ -25,11 +25,14 @@ def execute(self, payload):
         """
         Execute the payload and then clear the payload.
 
-        :return: Response
+        :return: string
```

Comment:
`response.json()` returns the json parsed, so it's not actually a string. I'd guess this should return a `list`, since we'll always have an array of actions (even if it's of size 1), right?

Reply:
It actually says the type is `dict` when I output the type.